File: typescript/tests/sortArray/sortArray.test.ts
import { sortArray } from "../../src/sortArray/sortArray";

describe('testing sort', () => {
    it('should return sorted array', () => {
        const nums = [2,7,11,4,5,3,9,17,14];
        const expected = [2,3,4,5,7,9,11,14,17];
        const actual = sortArray(nums);
        expect(expected).toEqual(actual);
    });
});

File: go/sortArray/sortArray_test.go
package main

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestSortArray(t *testing.T){
	var nums = []int {11,7,2,8,1,5,15,17,3}
	var actual = []int {2,3,5,7,8,11,15,17}
	expected := SortArray(nums)
	assert.Equal(expected, actual)
}